search-commons/src/test/resources/datasource_urls_ticket.json

```diff
@@ -16,8 +16,15 @@
   "MESSAGES_NOT=en+DOI": 19,
   "MODIFIED_DATE=1872-01-01,2024": 1,
   "ORGANIZATION_ID=https://api.dev.nva.aws.unit.no/cristin/organization/20754.0.0.0": 20,
```

Comment:
Even though ViewingScope is an alias of Organization_id, should there be a specific test where organization_id and excludeSubUnits are tested together? Or would it be unnecessary duplicate tests?

Reply:
2 lines futher down ;)

Reply:
Yes, I take it that it is unnecessary to duplicate the test with organization_id as well then. (It may not be apparent for everyone that viewingScope is an alias for organization_id)